pylintrc

```diff
@@ -9,7 +9,7 @@
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=missing-docstring,protected-access,invalid-name,global-statement,fixme,too-many-locals,redefined-builtin,broad-except,line-too-long,too-many-arguments,too-many-instance-attributes,redefined-outer-name,attribute-defined-outside-init,anomalous-backslash-in-string,too-many-statements,too-few-public-methods,redefined-argument-from-local,too-many-return-statements,too-many-function-args,redefined-argument-from-local,too-many-public-methods
+disable=missing-docstring,protected-access,invalid-name,global-statement,fixme,too-many-locals,redefined-builtin,broad-except,line-too-long,too-many-arguments,too-many-instance-attributes,redefined-outer-name,attribute-defined-outside-init,anomalous-backslash-in-string,too-many-statements,too-few-public-methods,redefined-argument-from-local,too-many-return-statements,too-many-function-args,redefined-argument-from-local,too-many-public-methods,inconsistent-return-statements, keyword-arg-before-vararg
```

Comment:
Are we sure these rules need to be ignored? I think this could be fixed in the code instead maybe?